modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/FolderBreadcrumb.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayBreadcrumb from '@clayui/breadcrumb';
+import {ClayButtonWithIcon} from '@clayui/button';
+import {ClayDropDownWithItems} from '@clayui/drop-down';
+import ClayToolbar from '@clayui/toolbar';
+import React from 'react';
+
+import SpaceSticker from './SpaceSticker';
+
+interface Props {
+	breadcrumbItems: BreadcrumbItem[];
+}
+
+interface BreadcrumbItem {
+	active?: boolean;
+	href?: string;
+	label: string;
+	onClick?: () => void;
+}
+
+const FolderBreadcrumb = ({breadcrumbItems}: Props) => {
```

Comment:
you can directly export the component: 
`export default function FolderBreadcrumb (...)`

Reply:
Done here [LPD-55752 Remove dropdown at the end of breadcrumb, refactor to function](https://github.com/liferay-content-management/liferay-portal/pull/6651/commits/ca37656b99e38090bbe4b3c720619fbcb0098c46)